tests/raw/test_basic_flow_stats.py

```diff
@@ -55,7 +55,9 @@ def test_basic_flow_stats(settings):
     # wait for flow metrics to be as expected
     while True:
         res = api.get_metrics(req)
-        if all([m.frames_tx == 10000 == m.frames_rx for m in res.flow_metrics]):
+        if all(
+            [m.frames_tx == 10000 == m.frames_rx for m in res.flow_metrics]
```

Comment:
does frames_tx work against athena ? I don't think it's supported, so just wondering if we're using this func in raw traffic test.

Reply:
> 
> 
> Please don't keep a dir called `demo_tests`, since these are real tests and we'll run them as part of cicd.
> 
> Put raw TCs in `raw` and device tests in device-specific dirs, e.g. `ip`, `bgp`, etc.

yeah moved to respective folder names

Reply:
lets address frames_tx later